### PR TITLE
(545) Fix: show assigned user names

### PR DESCRIPTION
**Pull request by mec**
## Changes

This behaviour was rebased away during #165 this work restores it and adds coverage to prevent it happening again.

I've refactored the feature spec that covers for two reasons:

1. I had broken it
2. I was finding it very hard to parse

https://trello.com/c/J5Xk2cD6

## Checklist

- [x] Attach this pull request to the appropriate card in Trello.
- [x] Update the `CHANGELOG.md` if needed.

### CODE REVIEW

**Review by jkempster34 — COMMENTED** [review]
Looks like the spec was rebased away too!

https://github.com/DFE-Digital/dfe-complete-conversions-transfers-and-changes/blob/737df5d7777394fab90f0da86e7195ff98818d25/spec/features/users_can_view_project_information_spec.rb#L37

**Comment by mec** [comment]
> Looks like the spec was rebased away too!
> 
> https://github.com/DFE-Digital/dfe-complete-conversions-transfers-and-changes/blob/737df5d7777394fab90f0da86e7195ff98818d25/spec/features/users_can_view_project_information_spec.rb#L37

Is that not this helper method that I updated:

https://github.com/DFE-Digital/dfe-complete-conversions-transfers-and-changes/blob/2f68a1a8426d03823870b6d5c9789062dac971e5/spec/features/users_can_view_project_information_spec.rb#L66

**Comment by jkempster34** [comment]
> > Looks like the spec was rebased away too!
> > https://github.com/DFE-Digital/dfe-complete-conversions-transfers-and-changes/blob/737df5d7777394fab90f0da86e7195ff98818d25/spec/features/users_can_view_project_information_spec.rb#L37
> 
> Is that not this helper method that I updated:
> 
> https://github.com/DFE-Digital/dfe-complete-conversions-transfers-and-changes/blob/2f68a1a8426d03823870b6d5c9789062dac971e5/spec/features/users_can_view_project_information_spec.rb#L66

Kind of - it looks like you have updated an older version of the helper method, and not the most recent method that captured the email addresses and full names

**Comment by mec** [comment]
@jkempster34 I went back to this and finally saw what you were trying to show me! 

For me the fact I wasn't seeing it speaks to me of the issue, it may be personal, but I found the feature spec really hard to read. I think it was the helper was overburdened and so the language was breaking down.

I've refactored this to include the link checks and hopefully make it a little more readable, I've removed the label check as for me that feels like testing Rails behaviour and I am not sure there is much value there.

Let me know what you think and sorry for the confusion, I blame working on the train! :)

**Review by jkempster34 — APPROVED** [review]
Think I preferred it more explicit 🤔 but happy to defer to your decision.

It definitely reads a bit easier 💯

**Comment by mec** [comment]
@jkempster34 okay, lets at least get the bug fixed and keep chipping away at improving the test suite for everyone! 👍
